elixir/apps/web/lib/web/live/clients/show.ex

```diff
@@ -135,7 +135,7 @@ defmodule Web.Clients.Show do
             </.link>
           </:col>
           <:empty>
-            <div class="text-center text-neutral-500 p-4">No authorizations to display</div>
+            <div class="text-center text-neutral-500 p-4">No activity to display</div>
```

Comment:
@jamilbk I left a comment on all table but we should update all of them

Reply:
👍